source/jobs/index.md

```diff
@@ -2,32 +2,32 @@
 layout: formatted-text
 title: Join the team
 ---
-<p class="intro-copy">There’ll soon be more seats around Convivio’s big table. It’s been a busy summer already and we’re planning to expand our team.</p>
+<p class="intro-copy">We are not currently hiring for any positions.</p>
 
-We prefer building relationships to the traditional ‘CV out of the blue’ approach — so we’d like to start hearing from people now who’d be interested in joining us in the coming months.
+We're always looking to hear from people who would be interested in working with us in the future. Drop us a line at <a href="mailto:jobs@weareconvivio.com">jobs@weareconvivio.com</a>
 
-If you’re looking for a new challenge in digital, then Convivio could be the perfect next move for your career.
+## What it's like to work at Convivio
 
-## What are we like?
+At Convivio we're not just remote, [we're free-range.](https://blog.weareconvivio.com/free-range-working-an-introduction-27eb178db97c) We don't have a central office and we all work from home. We encourage flexible working. [Read more about working from home at Convivio.](/jobs/working-from-home)
 
-We’re an enthusiastic, collaborative and friendly team who are quickly building a great agency to help our customers solve some big problems.
+We're a self-managing company, we all have a high-degreee of ownership over our work environment, process, and output.
 
-We focus on designing and building digital services — helping large public and private sector organisations move their core processes online.
+We're open and transparent, and believe in doing things right. We're the first digital agency to be [awarded the fair tax mark.](https://blog.weareconvivio.com/convivio-becomes-first-digital-agency-to-be-awarded-the-fair-tax-mark-881021825741)
 
-We work in very open ways, and often together with other partner agencies, or clients’ in-house teams.
+We offer unlimited holiday, an unlimited equipment budget, and unlimited training budget.
 
-We work in a distributed way, so you can be based anywhere (for now, within the EU), but you must be willing to travel regularly to London for key meetings and events.
+We focus on designing and building digital services — helping large public sector organisations improve how they serve the public. A lot of our clients are based in London.
```

Comment:
The word 'public' is used twice in close succession in the first sentence here. Can it be rephrased slightly?